.maintain/gitlab/check_polkadot.sh

```diff
@@ -73,7 +78,14 @@ then
     git fetch --depth 1 origin refs/pull/${pr_companion}/head:pr/${pr_companion}
     git checkout pr/${pr_companion}
   else
-    boldprint "no companion pr found - building polkadot:master"
+    if git fetch --depth 1 origin \
+      "$CI_MERGE_REQUEST_SOURCE_BRANCH_NAME":branch/"$CI_MERGE_REQUEST_SOURCE_BRANCH_NAME"
```

Comment:
I think that variable only is available if using gitlab merge requests. see https://docs.gitlab.com/ee/ci/variables/predefined_variables.html#variables-reference
github pull requests are mirrored as branches and their branches are not available by their names.

Reply:
I noticed